doc/_templates/sidebar-nav-bs.html

```diff
@@ -30,19 +55,14 @@
         </ul>
 
         {% if versions %}
-        <h5> Versions </h5>
-        <ul>
+        <h6> Versions </h6>
+        <select name="versions" id="versions" style="background:#F8F9FA;width:50%; border-radius:6px;border:1px solid #e5e5e5;font-family: inherit;font-size: inherit;padding: 0.4em 0.4em 0.4em 0.4em" onchange="onVersionChange()">
```

Comment:
That's... quite specific :) Are some generic bootstrap classes we can use instead?

Just asking, if the answer is no, that's fine too of course.

Reply:
Very specific! I'll have a look but I'm sure I'll find something more generic ;)

Reply:
Done.